packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -267,6 +267,16 @@ export default class extends Component {
                       : _('hostStatusDisabled')}
                   </td>
                 </tr>
+                {host.chipset_info.iommu !== undefined && (
+                  <tr>
+                    <th>{_('hostIommu')}</th>
```

Comment:
Don't forget to add a [tooltip](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/tooltip/index.js#L60) that says "PCI passthrough capable", as the issue suggests.